### PR TITLE
Fixed deployment.

### PR DESCRIPTION
**Pull request by cloud-badal**

### CODE REVIEW

File: charts/truefoundry-pipelines/templates/task-create-namespace-utils.yaml [hunk 1]
```diff
@@ -76,8 +76,8 @@ spec:
           hard:
             requests.cpu: "2"
             requests.memory: 2Gi
-            limits.cpu: "4"
-            limits.memory: 4Gi
+            limits.cpu: "10"
```
Comment (innoavator) [inline charts/truefoundry-pipelines/templates/task-create-namespace-utils.yaml:79]: This was earlier done based on some pricing constraints - can we override this from the sf-server?

**Review by innoavator — APPROVED** [review]

File: charts/truefoundry-pipelines/Chart.yaml [hunk 2]
```diff
@@ -1,6 +1,6 @@
 apiVersion: v2
 name: truefoundry-pipelines
-version: 0.0.8
+version: 0.0.29
```
Comment (innoavator) [inline charts/truefoundry-pipelines/Chart.yaml:3]: We can just make it 0.0.9 ?

Comment (cloud-badal) [reply charts/truefoundry-pipelines/Chart.yaml:3]: I have already deployed the middle version manually. So it won't work.

File: charts/truefoundry-pipelines/templates/task-create-namespace-utils.yaml (hunk 1 above)
Comment (cloud-badal) [reply charts/truefoundry-pipelines/templates/task-create-namespace-utils.yaml:79]: This need to be discussed in more detail.

**Review by innoavator — APPROVED** [review]

**Pull request merged**